### PR TITLE
Faq rbac aks dca

### PR DESCRIPTION
**Pull request by Simwar**
### What does this PR do?
It adds a faq article about the RBAC to apply when running the DCA on AKS deployed with our helm chart.

### Motivation
Many cases seen where this RBAC needs to be applied.

Preview: https://github.com/DataDog/documentation/blob/f326f6afb1477cd3fd527ac57ab520677ae99992/content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md

### CODE REVIEW

File: content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Does your Cluster Agent restart all the time on AKS?
```
Comment (CharlyF) [inline content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md]: Let's us a different phrasing.
"Cluster Agent or Agent restarting when using Azure Kubernetes Service (AKS)"

File: content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+It might be because you have deployed it with our helm chart and the readiness probe is killing it.
```
Comment (CharlyF) [inline content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md]: Maybe something like "Depending on how you deployed the manifest, you might be using readiness probes. On Azure AKS you will need extra permissions in the Cluster Role in order for the health endpoint to be reachable."

File: content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:healthz
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:authenticated
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:unauthenticated
```
Comment (CharlyF) [inline content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md]: Do we need the `system:unauthenticated` ?

File: content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md (hunk 2 above)
Comment (ruthnaebeck) [reply content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md]: Remove the word `will` - no future tense in docs:

"Depending on how you deployed the manifest, you might be using readiness probes. On Azure AKS you need extra permissions in the Cluster Role in order for the health endpoint to be reachable."

File: content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md (hunk 3 above)
Comment (cswatt) [reply content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md]: @CharlyF who should we ping for the answer to this?

Comment (Simwar) [reply content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md]: @xvello any insights?

Comment (xvello) [reply content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md]: This is due to the livenessprobe [defined here when custom metrics are enabled](https://github.com/helm/charts/blob/5a1ab776a91aa9119939f228009522d5e811bba7/stable/datadog/templates/cluster-agent-deployment.yaml#L93-L99)

The kubelet curls `dca-pod:443/healthz` with no authentication headers, so the request will fall into the `system:unauthenticated` group.

Comment (Simwar) [reply content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md]: Thanks! @cswatt can we merge this one now that the modifications are done?
Let me know if I need to do anything else.

Thanks!

**Pull request merged**